pallets/stellar-relay/src/default_weights.rs

```diff
@@ -1,41 +1,40 @@
 
-//! Autogenerated weights for `stellar_relay`
+//! Autogenerated weights for stellar_relay
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2023-05-24, STEPS: `100`, REPEAT: `10`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2023-06-14, STEPS: `100`, REPEAT: `10`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
-//! HOSTNAME: `Bs-MacBook-Pro.local`, CPU: `<UNKNOWN>`
+//! HOSTNAME: `Marcels-MBP`, CPU: `<UNKNOWN>`
 //! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
 
 // Executed Command:
 // ./target/release/spacewalk-standalone
 // benchmark
 // pallet
-// --chain
-// dev
-// --execution=wasm
-// --wasm-execution=compiled
-// --pallet
-// *
-// --extrinsic
-// *
+// --chain=dev
+// --pallet=stellar-relay
+// --extrinsic=*
 // --steps=100
 // --repeat=10
-// --output
-// pallets
+// --wasm-execution=compiled
+// --execution=wasm
+// --output=pallets/stellar-relay/src/default_weights.rs
+// --template=./.maintain/frame-weight-template.hbs
 
 #![cfg_attr(rustfmt, rustfmt_skip)]
 #![allow(unused_parens)]
 #![allow(unused_imports)]
+#![allow(missing_docs)]
 
-use frame_support::{traits::Get, weights::Weight};
-use sp_std::marker::PhantomData;
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use core::marker::PhantomData;
```

Comment:
Do you think this matters? It changed because the latest [template](https://github.com/paritytech/substrate/blob/2cc1667545f5e715fb94d492a2a2385ec8e00c7c/.maintain/frame-weight-template.hbs#L21) file uses `core::`